bucket/yacreader.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "9.5.0",
+    "homepage": "https://www.yacreader.com",
+    "description": "A comic reader and manager.",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/YACReader/yacreader/releases/download/9.5.0/YACReader-9.5.0-win64.exe",
+            "hash": "3dbe9d5da2584197ed8b0319b67363843a90148168db9d2465be3daabbcaf2dd"
+        },
+        "32bit": {
+            "url": "https://github.com/YACReader/yacreader/releases/download/9.5.0/YACReader-9.5.0-win32.exe",
+            "hash": "03ea3decfcfd5c0414ef0517b2a55e6a75fc4143cb633e28cbe7a141b5d802fb"
+        }
+    },
+    "innosetup": true,
+    "bin": "YACReader.exe",
+    "shortcuts": [
+        [
+            "YACReader.exe",
+            "YACReader"
+        ]
```

What is this executable for? It spawn some gui application

```suggestion
        ],
        [
            "YACReaderLibrary.exe",
            "YACReader Library"
        ]
```

---

It is a book manager.